dist/uninstall.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+echo "Uninstalling Steam Deck Plugin Loader and all plugins..."
+
+HOMEBREW_FOLDER=/home/deck/homebrew
+
+# Disable and remove services
+sudo systemctl disable --now plugin_loader.service > /dev/null
+sudo rm -f /home/deck/.config/systemd/user/plugin_loader.service
+sudo rm -f /etc/systemd/system/plugin_loader.service
+
+# Remove temporary folder if it exists from the install process
+rm -rf /tmp/plugin_loader
+
+# Remove folder structure
+sudo rm -rf ${HOMEBREW_FOLDER}
```

Comment:
```suggestion
# Cleanup services folder
sudo rm ${HOMEBREW_FOLDER}/services/PluginLoader
```
Before merging I think it would be best if the homebrew folder is preserved as if users are simply uninstalling to replace with a different version then it would be best to maintain the plugins folder etc.
I couldn't really get a nice safe ``rm`` invocation to clear the services folder nicely so this will need to be updated in the future as @WerWolv mentioned that there's the possibility of additional services so best to just purge specified contents of the folder for now.